static/js/admin.js

```diff
@@ -0,0 +1,322 @@
+    });
+    return;
+  }
+
+  $.post('/getAllCountryRegions', {country: countrySelection},
+      function(data, status) {
+        if (data) {
+          const regionData = data.regionData;
+          loadedRegionData = regionData;
+          const regionsParent = document.getElementById('regions');
+          regionsParent.innerText = '';
+          for (let i = 0; i < regionData.length; i++) {
+            const region = regionData[i];
+            regionsParent.innerHTML += '<div id="streetView' + i + '">' +
+              '<h1 class="title">' + region.region +
+              '&nbsp;<span style="font-size: 1rem">(Pin: <a href="https://maps.google.com/maps/search/' +
+              region.pin.lat + ',' + region.pin.lng + '" target="_blank">[' +
+              region.pin.lat + ', ' + region.pin.lng + '])' +
+              '</h1>';
+            regionsParent.innerHTML += '<br>';
+            for (let v = 0; v < region.streetViews.locations.length; v++) {
+              regionsParent.innerHTML +=
+                '<h2 class="subtitle" style="margin-left: 1rem">Street View ' +
+                v + '</span>&nbsp;&nbsp;&nbsp;<button class="button"' +
+                'onclick="showStreetView(' +
+                i + ', ' + v + ')"> Open Street View </button> <br>';
+            }
+            regionsParent.innerHTML += '</div>';
+          }
+          bulmaToast.toast({
+            message: '<h1>Regions loaded from DB.</h1>',
+            type: 'is-primary',
+            dismissible: true,
+            closeOnClick: true,
+            animate: {in: 'fadeIn', out: 'fadeOut'},
+          });
+        } else {
+          bulmaToast.toast({
+            message: '<h1>Error loading regions for country from DB.</h1>',
+            type: 'is-danger',
+            dismissible: true,
+            closeOnClick: true,
+            animate: {in: 'fadeIn', out: 'fadeOut'},
+          });
+        }
+      });
+}
+
+function showStreetView(regionIndex, streetViewIndex) {
+  panoramaRef = new google.maps.StreetViewPanorama(
+      document.getElementById(panoramaId), {
+        position: {lat:
+          loadedRegionData[regionIndex].streetViews
+              .locations[streetViewIndex]['_latitude'],
+        lng: loadedRegionData[regionIndex].streetViews
+            .locations[streetViewIndex]['_longitude']},
+        pov: {
+          heading: 180,
+          pitch: 10,
+        },
+      });
+  panoramaRef.setVisible(true);
+}
+
+function showGeneratedStreetView(region, streetViewIndex) {
+  panoramaRef = new google.maps.StreetViewPanorama(
+      document.getElementById(panoramaId), {
+        position: {
+          lat: generatedStreetViews[region][streetViewIndex].lat,
+          lng: generatedStreetViews[region][streetViewIndex].lng},
+        pov: {
+          heading: generatedStreetViews[region][streetViewIndex].h,
+          pitch: 10,
+        },
+      });
+}
+
+function removeGeneratedSV(region, streetViewIndex) {
```

Comment:
I keep thinking "SVG" when I see "SV". I suggest we standardize how we refer to street views in function names